bindings/python/CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ set(package_file ${CMAKE_BINARY_DIR}/packages/foundationdb-${FDB_VERSION}${not_f
 add_custom_command(OUTPUT ${package_file}
   COMMAND $<TARGET_FILE:Python3::Interpreter> -m ensurepip
   COMMAND $<TARGET_FILE:Python3::Interpreter> -m pip install --upgrade build
-  COMMAND $<TARGET_FILE:Python3::Interpreter> -m build &&
+  COMMAND $<TARGET_FILE:Python3::Interpreter> -m build --sdist &&
```

Comment:
Why --sdist now and it wasn't needed before? What sort of bundle did we make before?  Binary and source?

Reply:
Without this option, we are generating python wheel files, which is not needed. We only publish tar.gz packages.

Reply:
ack